manifests/metallb-frr-WIP.yaml

```diff
@@ -0,0 +1,620 @@
+    # the normal daemon command is added to this at the end.
+  vtysh.conf: ""
+  zebra.conf: |+
+    ! -*- zebra -*-
+    !
+    ! zebra sample configuration file
+    !
+    hostname Router
+    password zebra
+    enable password zebra
+    !
+    ! Interface's description.
+    !
+    !interface lo
+    ! description test of desc.
+    !
+    !interface sit0
+    ! multicast
+
+    !
+    ! Static default route sample.
+    !
+    !ip route 0.0.0.0/0 203.181.89.241
+    !
+
+    !log file zebra.log
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: metallb
+    component: speaker
+  name: speaker
+  namespace: metallb-system
+spec:
+  selector:
+    matchLabels:
+      app: metallb
+      component: speaker
+  template:
+    metadata:
+      annotations:
+        prometheus.io/port: '7472'
+        prometheus.io/scrape: 'true'
+      labels:
+        app: metallb
+        component: speaker
+    spec:
+      volumes:
+        - name: frr-sockets
+          emptyDir: {}
+        - name: frr-startup
+          configMap:
+            name: frr-startup
+        - name: frr-conf
+          emptyDir: {}
+        - name: reloader
+          emptyDir: {}
+      initContainers:
+        # Copies the initial config files with the right permissions to the shared volume.
+        - name: cp-frr-files
+          securityContext:
+            runAsUser: 100
+            runAsGroup: 101
+          image: frrouting/frr:v7.5.1
+          command: ["/bin/sh", "-c", "cp -rLf /tmp/frr/* /etc/frr/"]
+          volumeMounts:
+            - name: frr-startup
+              mountPath: /tmp/frr
+            - name: frr-conf
+              mountPath: /etc/frr
+        # Copies the reloader to the shared volume between the speaker and reloader.
+        - name: cp-reloader
+          image: quay.io/metallb/speaker:main
+          command: ["/bin/sh", "-c", "cp -f /frr-reloader.sh /etc/frr_reloader/"]
+          volumeMounts:
+            - name: reloader
+              mountPath: /etc/frr_reloader
+      containers:
+        - name: frr
+          securityContext:
+            capabilities:
+              add: ["NET_ADMIN", "NET_RAW", "SYS_ADMIN", "NET_BIND_SERVICE"]
+          image: frrouting/frr:v7.5.1
+          env:
+            - name: TINI_SUBREAPER
+              value: "true"
+          volumeMounts:
+            - name: frr-sockets
+              mountPath: /var/run/frr
+            - name: frr-conf
+              mountPath: /etc/frr
+        - name: reloader
+          image: frrouting/frr:v7.5.1
+          command: ["/etc/frr_reloader/frr-reloader.sh"]
+          volumeMounts:
+            - name: frr-sockets
+              mountPath: /var/run/frr
+            - name: frr-conf
+              mountPath: /etc/frr
+            - name: reloader
+              mountPath: /etc/frr_reloader
+        - args:
+            - --port=7472
+            - --config=config
+            - --log-level=info
+          env:
```

Comment:
Need to set BGP type to frr, otherwise speaker will default to native.
```
- name: METALLB_BGP_TYPE
  value: frr
```

Reply:
will be as part of the actual integration PR.